### ISSUE TITLE
Errors after installation (Testbed not running)

### ISSUE THREAD

**Issue by Arian-IDSA** [International-Data-Spaces-Association/IDS-testbed#67]
Hi All,

Testbed is not running after installation. I have changed my approach and tried on separated VMs, still no luck.
- Ubuntu version: 20.04. 
- OS and packages are updated. 
Installation was run as super user. 

Screenshot: 
<img width="534" alt="testbed-not-running-Arian" src="https://user-images.githubusercontent.com/97449699/168035110-622b04ce-fc98-4207-add0-57b8a5ee37ef.PNG">

Full installation output: 
[testbed-installation.txt](https://github.com/International-Data-Spaces-Association/IDS-testbed/files/8676618/testbed-installation.txt)

I have received an email from [Iva](https://github.com/ivakra) @ivakra and she has faced the same issue:
![errorTestbed_04052022](https://user-images.githubusercontent.com/97449699/168035910-c4564d36-08e1-4320-9318-0ac17f40734e.jpg)

**Comment by sebplorenz** [comment]
Cannot reproduce. Did you check out only the install.sh file? If yes, then you should check out the whole repository and try again.

**Comment by Arian-IDSA** [comment]
Thanks a lot Sebastian! Yes, only install.sh. Will checkout the whole repo and will try again soon, but if that's the reason, I think we should either change it or add it to the doc.

**Comment by jfernandezsqs** [comment]
Hello Arian and @ivakra,
I am not able to reproduce your error at the execution of the `install.sh` script.
However, I have changed the documentation of the file 'InstallationGuide.md' and I think that the changes could fix your error.
Can you please follow the steps included [Target View: Preconfigured testbed](https://github.com/International-Data-Spaces-Association/IDS-testbed/blob/Documentation_Update/InstallationGuide.md#target-view-preconfigured-testbed) and see if it works with the commands that are mentioned there.

**Comment by ivakra** [comment]
Thank you @sebplorenz and @jfernandezsqs! 
I executed the steps as specified in the [Target View: Preconfigured testbed](https://github.com/International-Data-Spaces-Association/IDS-testbed/blob/Documentation_Update/InstallationGuide.md#target-view-preconfigured-testbed) and deployment was successful :)
Following is description of the steps and a couple of comments:
1. Regarding the software requirements: I actually started from a clean VM - nothing but the required OS installed (and it worked)
2.  lsb_release -a
When I copied and pasted the "$lsb_release -a" command, I actually received an error because of the extra '$' sign copied in front of the command.
3. sudo apt-get update
4. sudo apt-get upgrade
5. sudo apt install git
6. git clone https://github.com/International-Data-Spaces-Association/IDS-testbed.git
7. cd IDS-testbed
8. sudo ./install.sh
I didn't try executing the install script with user privileges. 
I reviewed the output of the execution of the install script and I noticed several warnings - I am attaching the file with the output in case you want to chase them
[output_170522.txt](https://github.com/International-Data-Spaces-Association/IDS-testbed/files/8709225/output_170522.txt)
